lib/compat/wordpress-6.3/block-template-utils.php

```diff
@@ -49,7 +49,7 @@ function gutenberg_get_default_block_template_types( $default_template_types ) {
 		$default_template_types['singular'] = array(
 			'title'       => _x( 'Single Entries', 'Template name', 'gutenberg' ),
 			'description' => __(
-				'Displays any single entry, such as a post or a page. This template will serve as a fallback when a more specific template (e.g., Single Post, Page, or Attachment) cannot be found.',
+				'Displays any single entry, such as a post or a page. This template will serve as a fallback when a more specific template (e.g. Single Post, Page, or Attachment) cannot be found.',
```

Comment:
We'll also have to change the corresponding strings in the E2E tests.

See the failing test [below](https://github.com/WordPress/gutenberg/pull/52848/files#file-packages-e2e-tests-specs-site-editor-settings-sidebar-test-js-L84).

Reply:
Thanks @ramonjd  - is that something I can do within this repo - or is it tooling that is specific elsewhere?

Reply:
> is that something I can do within this repo 

Yes for sure. The files exist in the Gutenberg repo.

 If you can do a "find in files" search for the modified strings you should be able to hunt them down.

Here's the one: https://github.com/WordPress/gutenberg/blob/500c09f03adc519264c4ac724e0b6f5d0b3df09d/packages/e2e-tests/specs/site-editor/settings-sidebar.test.js#L84-L84

I think that's the only one, but it wouldn't hurt to check